Add beta notice for image uploads

johnwcallahan opened the pull request:
## Description

This PR adds a beta notice to the Upload Image tab, and updates the container padding on both image creation tabs:

![Screen Shot 2021-04-20 at 5 30 46 PM](https://user-images.githubusercontent.com/16911484/115466730-79133880-a1fe-11eb-93d1-682b731f4252.png)


## How to test

Visit localhost:3000/images/create. View both tabs to check the container spacing. Look near the "Generate URL" button to see the beta notice.

I'm open to wordsmithing here. I couldn't exactly figure out what to call this feature... we don't say "Machine Images" anywhere else on this page. The grammar might be a bit weird too... "Image Uploads **are** currently in beta...". Would love to hear your thoughts.

matthewbartholomew commented:
A few thoughts:
1. Plural vs singular, I am leaning to singular because Image Uploads is a proper noun. Both ways have their own awkwardness, tho.
2. It might be more consistent and noticeable to put the notice at the top.
3. Maybe we use the Beta tag instead of "Note" in bold? Just a thought.
4. Have more space above the button would help the visual structure
![Beta message 2](https://user-images.githubusercontent.com/16086141/115588304-97c80c80-a29c-11eb-9441-39261e99fe6a.png)
![Beta message](https://user-images.githubusercontent.com/16086141/115588305-9860a300-a29c-11eb-811f-5994318b3f87.png)

Jskobos commented on packages/manager/src/features/Images/ImageUpload.tsx:
```diff
@@ -116,7 +118,18 @@ export const ImageUpload: React.FC<Props> = (props) => {
+        <Typography style={{ marginTop: 16 }}>
+          <strong>Note:</strong> Image Uploads are currently in beta and are
+          subject to the terms of the{' '}
+          <ExternalLink
```
We don't really need the externallink component anymore, if you use src/components/link and the URL starts with https it does the same thing.

Jskobos reviewed (approved):
Looks good, I know Matthew asked for a Beta chip, I was going to suggest putting that in the tab title as we've done elsewhere in the past (maybe?)

tiffwong reviewed (approved):
Looks good and I like the idea of using the beta tag somewhere

johnwcallahan commented:
@matthewbartholomew addressed your feedback:

<img width="1014" alt="Screen Shot 2021-04-21 at 4 44 25 PM" src="https://user-images.githubusercontent.com/16911484/115618550-dbcd0880-a2c0-11eb-899d-b82697e67fd0.png">
